dev/NavigationView/NavigationView.cpp

```diff
@@ -1741,24 +1743,9 @@ void NavigationView::UpdatePaneButtonsWidths()
         }
         return CompactPaneLength();
     }();
-
-    if (auto&& backButton = m_backButton.get())
-    {
-        backButton.Width(newButtonWidths);
-    }
-    if (auto&& paneToggleButton = m_paneToggleButton.get())
-    {
-        paneToggleButton.MinWidth(newButtonWidths);
-        if (const auto iconGridColumnElement = paneToggleButton.GetTemplateChild(c_paneToggleButtonIconGridColumnName))
-        {
-            if (const auto paneToggleButtonIconColumn = iconGridColumnElement.try_as<winrt::ColumnDefinition>())
-            {
-                auto width = paneToggleButtonIconColumn.Width();
-                width.Value = newButtonWidths;
-                paneToggleButtonIconColumn.Width(width);
-            }
-        }
-    }
+ 
+    templateSettings->PaneButtonWidth(newButtonWidths);
+    templateSettings->LatestPaneButtonWidth(newButtonWidths - 8);
```

Comment:
>LatestPaneButtonWidth [](start = 22, length = 21)

This is unfortunate that we need to create a new API because we cannot do x - 8 in markup :/

Reply:
Could we add a new converter allowing subtraction/addition to values similar to the TabView inset corner radii case where we added multiplication to thickness?

Reply:
Those are the two options available. In the past we have resisted making an AdditionConverter as its a bit too.. generic? and the cost of a template setting has historically been viewed as low.